static/app/views/replays/types.tsx

```diff
@@ -1,6 +1,10 @@
 export type Replay = {
+  count_if_event_type_equals_error: number;
+  'equation[0]': number;
   eventID: string;
   id: string;
+  max_timestamp: string;
+  min_timestamp: string;
```

Comment:
We shouldn't have these in here. The response we are getting from our two different queries are going to be different, so we should have two different types.

I would define a ReplayDurationAndErrors type in the `replaysTable` file since it probably won't be used as a shared type.